docs/pages/tooltip.js

```diff
@@ -822,7 +837,11 @@ Toast provides feedback on an interaction. One example of Toast is the confirmat
 }
 
 export async function getServerSideProps(): Promise<{| props: {| generatedDocGen: DocGen |} |}> {
+  const generatedDocGen = await docgen({ componentName: 'Tooltip' });
+
+  generatedDocGen.description = generatedDocGen.description.split('\n')[0];
```

Comment:
This is a bit brittle, as it relies on the comment formatting rather than the content. Can we instead take an approach similar to how we read default values in comments, such that we can display only the description on the docs, and the description + banner content in IDEs?

Reply:
Let me see... the problem is that we want the Alert text on the description... I'll try to see if I can parse somehow.